api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirTaskDaoImpl.java

```diff
@@ -78,7 +78,8 @@ public FhirTask getTaskByUuid(String uuid) {
 		// TODO: Handle optional params
 		// Task.basedOn
 		if (validReferenceParam(basedOnReference)) {
-			criteria.createAlias("basedOnReferences", "bo").add(Restrictions.eq("bo.reference", basedOnReference.getIdPart()))
+			criteria.createAlias("basedOnReferences", "bo")
+			        .add(Restrictions.eq("bo.reference", basedOnReference.getIdPart()))
 			        .add(Restrictions.eq("bo.type", basedOnReference.getResourceType()));
 		}
```

Comment:
Is this part of the optimization? If not, You should remove it

Reply:
These are formatting changes applied by mvn clean install. I believe they should be part of the PR too.

Reply:
I'd take these out of the PR. It's going to be changed in a different commit and will just cause a conflict later.